### ISSUE TITLE
bulk scripts don't appear in audit log 

### ISSUE THREAD

**Issue by bbrendon** [amidaware/tacticalrmm#1773]
See screenshot. They are missing. This was discussed previously with no decision. https://github.com/amidaware/tacticalrmm/issues/995

After looking at this more, I think this would be of serious concern to anyone looking to use the audit feature in production.  There is no way to know if a script ran on a host because the log seems to just say "servers". Basically it throws out the audit log for script auditing because there is no way to know if a script was run on a host. I suppose you could extrapolate from the database the time a host was added or moved to the group and compare it to when the bulk audit was run but I don't think that is reasonable approach. 

<img width="1123" alt="image" src="https://github.com/amidaware/tacticalrmm/assets/6364477/4a265193-3a3b-49c3-99ee-ccc9f5a5ac47">

**Comment by silversword411** [comment]
Yes, we need to go thru all the places and add more audit recording stuff. 

I actually looked at this over the weekend with wh1te, will try and see about tackling at least some of the places.
